openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/mysqlConnection.json

```diff
@@ -44,6 +44,7 @@
     "authType": {
       "title": "Auth Configuration Type",
       "description": "Choose Auth Config Type.",
+      "mask":  true,
```

Comment:
do we need the `mask` at this level?

Reply:
it works without mask as well, i was checking at other connection sons we are adding it similarly
if doesn't make much sense we can remove it

Reply:
@pmbrull lmk wdyt